spec/intl.html

```diff
@@ -25,26 +25,25 @@ <h1><a href="https://tc39.es/ecma402/#sec-time-zone-names">Time Zone Names</a></
     </ins>
 
     <p>
-      The ECMAScript 2021 Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database. Their canonical form is the corresponding Zone name in the casing used in the IANA Time Zone Database.
+      The ECMAScript 2021 Internationalization API Specification identifies time zones using the Zone and Link names of the IANA Time Zone Database. Their canonical form is the corresponding Zone name in the casing used in the IANA Time Zone Database <ins>except as specifically overridden by CanonicalizeTimeZoneName</ins>.
```

Comment:
This corrects a current mistake in ECMA-402, which overrides "Etc/UTC" and "Etc/GMT" (which are canonical names for distinct time zones) with "UTC" (which is an alias of the former).